cedar-policy/CHANGELOG.md

```diff
@@ -148,6 +155,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   format. (#734, resolving #681)
 
 ## [3.1.2] - 2024-03-29
+Cedar Language Version: 3.2.0
 
```

Comment:
It seems strange that the Rust create is a patch version update but the language is a minor version. Why is that?

Reply:
I don't recall the exact reasoning, but I think RFC#57 (which allows multiplication of arbitrary expressions) was needed to fix some weird parsing behavior, so we decided it was a "bug fix". From a language standpoint, it seems more like a "minor change" to me since it expands the class of parseable Cedar programs.

Definitely a grey area though... open to counter arguments.